src/components/Filter/Filter.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import Filter from './Filter';
+
+describe('Filter component', () => {
+  it('Should be show filter component', () => {
+    const filters: string[] = ['CSS', 'Frontend'];
+
+    const filter = render(<Filter filters={filters} />);
+
+    expect(filter).toBeTruthy();
```

Comment:
Shouldn't you test using the `innerHTML` as in the second test? 

I'm not sure if returning null from the component makes the component to
still be truthy, since it's being rendered as null (but rendered).

Reply:
The logic was removed, according to your recommendation, it will never pass undefined, null, etc.